README.md

```diff
@@ -8,26 +8,26 @@
 <img src="https://github.com/Jigsaw-Code/outline-brand/blob/main/assets/powered_by_outline/color/logo.png?raw=true" width=400pt />
 </p>
 
-The Outline SDK helps developers:
+> ⚠️ **Warning**: This code is in early stages and is not guaranteed to be stable. If you are
+> interested in integrating with it, we'd love your feedback.
```

Comment:
How does one give us feedback?

Reply:
Maybe via this GitHub repository?

Reply:
If we want them to log issues with feedback and suggestions, we could link to here: https://github.com/Jigsaw-Code/outline-internal-sdk/issues/new
Note that we don't have template set up for this repo yet so it might be a free-for-all, but we can do that in the coming weeks to have some structure for feature requests, bugs, and general feedback

Reply:
Added the link to the doc,